drivers/sensor/renesas/rrh46410/Kconfig

```diff
@@ -0,0 +1,24 @@
+# RRH46410 air quality sensor
+
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config RRH46410
+	bool "RRH46410 sensor"
+	default y
+	depends on DT_HAS_RENESAS_RRH46410_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_RENESAS_RRH46410),i2c)
+	depends on GPIO
+
+	help
+	  Enable driver for RRH46410 TVOC sensor.
+
+if RRH46410
+
+config RRH46410_UART
+	bool "rrh46410 uart"
+	default n
+	select UART
+	depends on UART_INTERRUPT_DRIVEN
```

Comment:
Remove this and update the previous Kconfig to use `dt_compat_on_bus` for serial too.